pkg/util/chanmanager/chanmanager.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Channel Manager keeps track of multiple channels
+package chanmanager
+
+import (
+	"fmt"
+	"sync"
+)
+
+type ChanManager interface {
```

Comment:
Please add a godoc comment. Also, can we name this for the service it provides instead of for the implementation it uses? Like, OperationManager or something?